website/static/css/pages/project-page.css

```diff
@@ -27,6 +27,39 @@
     content: "";
 }
 
+#contributors p {
+    margin: 0;
+}
+#contributors-list {
+    position: relative;
+    overflow: hidden;
+}
+#contributor-toggle {
+    border: 1px solid #EEE;
+    background-color: #F8F8F8;
+    height: 15px;
+    line-height: 10px;
+    color: #888;
+}
+#contributor-toggle:hover {
+    background-color: #F5F5F5;
+    color: #666;
```

Comment:
Have you considered adding `cursor: hand` to this rule, to indicate that the element is clickable?